docs/releases/versions/version-v10.0.0.md

```diff
@@ -38,7 +38,7 @@ When a community field is required, it will be also displayed in the form to [cr
 
 ### Administration panel
 
-The new administration panel provides a modern, clean and easy to use interface. It allows to manage the repository's settings from the web interface without requiring technical knowledge. It is also allow customising existing views or add new ones.
+The new administration panel provides a modern, clean and easy to use interface. It allows to manage the repository's settings from the web interface without requiring technical knowledge. It also allows customising existing views or add new ones.
```

Comment:
```suggestion
The new administration panel provides a modern, clean and easy to use interface. It allows to manage the repository's settings from the web interface without requiring technical knowledge. It also allows customising existing views or adding new ones.
```

Perhaps something to discuss in the future by the Docs/On-boarding team: do we use "English" or "American" spelling for words e.g., "customising" or "customizing"?